src/components/DeploymentSchedulesFieldset.vue

```diff
@@ -6,7 +6,7 @@
 
     <ScheduleFormModal v-if="deployment.can.update" :active="null" :schedule="null" :job-variables="{}" @submit="createSchedule">
       <template #default="{ open }">
-        <p-button size="sm" icon="PlusIcon" @click="open">
+        <p-button size="sm" icon="PlusIcon" :disabled="deployment.disabled" @click="open">
```

Comment:
Should schedules not be modifiable if a deployment is disabled? I'd sort of expect I could modify it all I want, it just wouldn't submit work

Reply:
I think flipping a schedule is an action that a user expects to have a result. If I flip the toggle and nothing happens, that'd be pretty confusing if I'm not paying attention to the disabled toggle.

Reply:
hm ok that makes sense - easy to walk back if it's confusing as well